crates/libzkp/src/verifier/euclidv2.rs

```diff
@@ -1,10 +1,12 @@
 use super::{ProofVerifier, TaskType, VKDump};
 
-use anyhow::Result;
+use eyre::Result;
```

Comment:
one question: the filed name is still `euclidv2`, shall we change it to `feynman`?